activitypub/apmodels/webfinger.go

```diff
@@ -43,6 +45,16 @@ func MakeWebfingerResponse(account string, inbox string, host string) WebfingerR
 				Type: "text/html",
 				Href: accountIRI.String(),
 			},
+			{
+				Rel:  "avatar",
```

Comment:
Should be `http://webfinger.net/rel/avatar` according to https://webfinger.net/rel/.

Reply:
Okay